src/main/java/org/folio/search/utils/SearchResponseUtils.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.search.utils;
+
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.folio.search.domain.dto.FolioCreateIndexResponse;
+import org.folio.search.domain.dto.FolioIndexOperationResponse;
+import org.folio.search.domain.dto.FolioIndexOperationResponse.StatusEnum;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class SearchResponseUtils {
```

Comment:
Do you mind to make the class more specific: for example rename to `IndecesResponseBuilder` or something like that?

Reply:
This a class with some factory methods to create response as one liner in repository, maybe later we will deprecate and remove it. Now I've renamed it to `SearchResponseHelper` which is more clear than previous name